### PR TITLE
fixed wrong condition when checked current version of the cdk

### LINKED ISSUE #49: postinstall wrong message
**Describe the bug**
When installing dependencies after Mira was added, it prompts to install cdk 1.49.1 but 1.49.1 is already installed.

**To Reproduce**
Add Mira as a dependency to you project
run `npm install`

**Expected behavior**
Shows warning about wrong CDK version if not installed

**Additional context**

### PR DESCRIPTION
**Pull request by zimny**
resolves #49

### CODE REVIEW

File: scripts/postinstall.js [hunk 1]
```diff
@@ -3,31 +3,32 @@ const colors = require('colors')
 // eslint-disable-next-line
 const fs = require('fs')
 
-const supportedCdkVersion = '"1.49.1"'
+const supportedCdkVersion = '1.49.1'
+const supportedCdkVersionString = `"${supportedCdkVersion}"`
 
 const mandatoryDepsMessage = `
 Please make sure to to include the following dependencies to you project:
```
Comment (irelandm) [inline scripts/postinstall.js:10]:
```suggestion
Please make sure to include the following dependencies for your project:
```

File: scripts/postinstall.js [hunk 2]
```diff
@@ -3,31 +3,32 @@ const colors = require('colors')
 // eslint-disable-next-line
 const fs = require('fs')
 
-const supportedCdkVersion = '"1.49.1"'
+const supportedCdkVersion = '1.49.1'
+const supportedCdkVersionString = `"${supportedCdkVersion}"`
```
Comment (irelandm) [inline scripts/postinstall.js:7]: I'm wondering about the reason for this extra variable?
Have you tried removing the extra set of quotes, e.g.
`const supportedCdkVersion = '1.49.1'` 
(on the previous commit, not this one)

Comment (zimny) [reply scripts/postinstall.js:7]: done

File: scripts/postinstall.js (hunk 1 above)
Comment (zimny) [reply scripts/postinstall.js]: done

Comment (irelandm) [reply scripts/postinstall.js]: Close! check that last 'you' -> 'your'!

**Pull request merged**